deltacat/compute/compactor/compaction_session.py

```diff
@@ -366,6 +407,18 @@ def _execute_compaction_round(
     logger.info(
         f"Got {total_hb_record_count} hash bucket records from hash bucketing step..."
     )
+    peak_memory_during_hb = max(
+        [hb_result.peak_memory_usage_bytes for hb_result in hb_results]
+    )
+
+    telemetry_time_hb = sum(
+        [hb_result.telemetry_time_in_seconds for hb_result in hb_results]
+    )
+
+    compaction_audit.set_input_records(total_hb_record_count.item())
+    compaction_audit.set_peak_memory_used_bytes_by_hash_bucketing(
```

Comment:
Renamed to `set_peak_memory_used_bytes_per_hash_bucket_task`. Since this goes to data warehouse, a single column for each task is easy to analyze instead of a dictionary.